content/Monomod/MonoMod__ModTemplate.cs

```diff
@@ -3,12 +3,14 @@
 #endif
 using BepInEx;
 using BepInEx.Logging;
-#if (!UseHookGen)
+#if (UseHookGen)
+using System.Reflection;
+using MonoMod.RuntimeDetour.HookGen;
+#else
 using System.Collections.Generic;
 using MonoMod.RuntimeDetour;
-using HarmonyLib;
 #endif
-using MonoMod._ModTemplate.Hooks;
+using MonoMod__ModTemplate.Patches;
```

Comment:
Please make sure it's `MonoMod._ModTemplate` - has to do with special formatting for namespaces vs classes